content/docs/user-guide/assets/asset-processor/asset-cache-server.md

```diff
@@ -0,0 +1,105 @@
+---
+linkTitle: Asset Cache Server Mode
+title: Asset Cache Server
+description: Enable a feature to reduce asset builds for a team.
+weight: 200
+toc: true
+---
+
+The **Asset Server Cache** mode is a feature allows an **Asset Processor** to cache product asset files to a remote folder so that other team members can skip long asset processing times.
+
+The **Asset Server Cache** (ACS) mode lets the **Asset Processor** fetch pre-processed products of a job from an asset server cache instead of processing it locally. It does this by using a shared folder where the **Asset Processor** writes out product asset archive ZIP files for a processed source asset job. The other **Asset Processor** clients retrieve these ZIP files and unzip the product assets instead of processing the source asset from scratch.
+
+# Setting Up Asset Processor in Asset Cache Server Mode
+
+The **Asset Cache Server** (ACS) mode has three steps to set up the system so that developers can use it.
+
+{{< note >}}
+The next sections assume the root folder to be `T:/o3de` but it can be in any folder or it can run on Linux.
+{{< /note >}}
+
+## Create the Transfer Folder
+
+The team will need to create a shared folder that all the team members will be able to access over the network. The tech should work between Windows and Linux, but the cacheServerAddress string only accepts a single string such as `T:/o3de/Transfer/Cache` that all the clients are meant to read from. It is possible to be clever about having an **Asset Processor** in ACS mode write to one configured `cacheServerAddress` string via Windows and having another ACS retrieve from a Samba link on Linux; but that will be a challenge the team will need to solve.
```

Comment:
Minor style changes. Also rewording the last sentence a bit to underscore it's up to the team's IT provider to solve the issue.

```suggestion
## Create the transfer directory

The team needs to create a shared directory that all the team members can access over the network. ACS should work between Windows and Linux, but the `cacheServerAddress` string only accepts a single string such as `T:/o3de/Transfer/Cache` that all the clients are meant to read from. It's possible to have an Asset Processor in ACS mode write to one configured `cacheServerAddress` string via Windows and have another ACS retrieve from a Samba link on Linux. Your team's IT manager can provide a solution for this scenario.
```